### ISSUE TITLE
Выводит все товары.

### ISSUE THREAD

**Issue by Vladusha94** [darkrain/woo-export-yml#1]

**Comment by Vladusha94** [comment]
Доброго времени суток. Скачал и установил Ваш плагин. Провел настройку. Выбрал нужные категории и на жал кнопку "Обновить все товары". Плагин сгенерировал файл, но выдал в него все товары, со всех категорий. Что не так?   Wordpress 4.2.2, Версия woocomerce 2.3.13. И еще проблема, создал второй источник, а когда нажимаешь кнопочку Удалить, то страница перезагружается а источник остается.

**Comment by darkrain** [comment]
Проверьте, вы случайно не выбрали в этом списке - Все. Если он отмечен, то будет выводиться все

**Comment by Vladusha94** [comment]
Нет, я уже заметил, если простые товары, то плагин работает замечательно, а если на сайте вариативный товар, то плагин работает не правильно. Т.е если на сайте только одни простые товары, то какую я бы категорию не выбрал он выводит все правильно, но как только я добавляю вариативный тавар, то он начинает не зависимо от категории выводить все товары.

**Comment by Vladusha94** [comment]
В раздел "categories"
"/categories"  он вставляет те категории которые я задаю, а в раздел "offers" вставляет все материалы со всех категорий. Но при условии что на сайте Вариативный товар!!!

**Comment by darkrain** [comment]
Где то баг, я могу долго решать вопрос, но если кто то решит отправьте мне пулл реквест, чтобы никто больше не мучался
